app/src/main/java/org/mozilla/focus/search/MultiselectSearchEngineListPreference.java

```diff
@@ -31,8 +31,9 @@ protected int getItemResId() {
 
     @Override
     protected void updateDefaultItem(CompoundButton defaultButton) {
-        // Hide default engine so it can't be removed
-        defaultButton.setVisibility(View.GONE);
+        defaultButton.setClickable(false);
+        // Use state_activated works because state_enabled doesn't work in state list.
```

Comment:
What does this  comment mean? Is `.setActivated` really necessary? I'd think you'd just need to set it not clickable.

Reply:
The reason for 'activated' is so that we can have use a statelistdrawable - there isn't a state_clickable, and for some reason state_enabled was just not working (perhaps it needed a default drawable state).

https://developer.android.com/reference/android/graphics/drawable/StateListDrawable.html

Reply:
Can you make your comment sound more like this explanation? :)